From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 13 Oct 2012 01:05:40 +0200
Subject: Properly parse (unsigned) long long

This fixes problems on platforms where sizeof(long) != sizeof(long long)
like ia32.
---
 python/generator.py |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/generator.py b/python/generator.py
index a98a894..ced7e41 100755
--- a/python/generator.py
+++ b/python/generator.py
@@ -259,8 +259,8 @@ py_types = {
     'double':  ('d', None, "double", "double"),
     'unsigned int':  ('i', None, "int", "int"),
     'unsigned long':  ('l', None, "long", "long"),
-    'long long':  ('l', None, "longlong", "long long"),
-    'unsigned long long':  ('l', None, "longlong", "long long"),
+    'long long':  ('L', None, "longlong", "long long"),
+    'unsigned long long':  ('L', None, "longlong", "long long"),
     'unsigned char *':  ('z', None, "charPtr", "char *"),
     'char *':  ('z', None, "charPtr", "char *"),
     'const char *':  ('z', None, "constcharPtr", "const char *"),
